registry.json

```diff
@@ -4533,5 +4533,19 @@
     "telegram": "CrypTopo",
     "operatorName": "Alfred",
     "github": "avilsmeier"
+  },
+   {
+    "ticker": "ADACT",
+    "poolId": "22ab39540db22349b1934f5dcb7788440c33709ea9fdac07fb343395",
+    "social": {
+     "website": "https://adacapital.io",
+     "twitter": "ada_capital",
+     "telegram": "adacapital",
+     "youtube": "https://www.youtube.com/channel/UCnZQvrAsqteYLpmEWb418yA",
+     "medium": "ada-capital"
+    },
+    "telegram": "adacapital",
```

Comment:
Please change this to be the username of an individual and not a group or channel

Reply:
Which tag in particular should be that of an individual?

Reply:
The telegram field.

Reply:
Ok got rid of the telegram field. Thanks for the heads-up. Thomas